### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by HowJMay**
## Status
**READY**

## Requires Backporting

NO  

## Migrations
NO

## Additional comments
No

## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Changelog updated
- [ ] Backported

## Steps to test or reproduce
Outline the steps to test or reproduce the PR here.

### CODE REVIEW

**Comment by gilles-peskine-arm** [comment]
Thanks! We acknowledge all contributions in the changelog, even fixes to comments. Please create a changelog entry file under `ChangeLog.d`. See `ChangeLog` for examples of how to word entries. This would be under “Changes”.

**Comment by HowJMay** [comment]
> Sorry for the delay here but the CI failed because your change log entry requires a newline character at the end.

Fixed. Thank you for explaining

**Comment by gilles-peskine-arm** [comment]
More simply, the test that's failing is
```
tests/scripts/check_files.py
```

**Comment by HowJMay** [comment]
Sorry for the mistake

File: ChangeLog.d/fix-typo.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Bugfix
+   * There is typo in the example code which locates in
```
Comment (daverodgman) [inline ChangeLog.d/fix-typo.txt:2]:
```suggestion
   * There is a typo in the example code located in
```

File: ChangeLog.d/fix-typo.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Bugfix
```
Comment (daverodgman) [inline ChangeLog.d/fix-typo.txt:1]:
```suggestion
Changes
```

**Review by daverodgman — COMMENTED** [review]
Hi, it looks like the whitespace error is now corrected, thanks.

However - as @gilles-peskine-arm mentions above, we'd normally file this kind of thing under "Changes" rather than "Bugfixes". I've also noted a couple of very minor phrasing issues. Also, ideally the first line of the commit message should start with a capital ("Fix typo" not "fix typo"), and could be a little more informative - I suggest e.g. "Fix typo in comment".

Please could you take a look at these changes? I don't mind adding a patch to your PR to fix these, but I'll give you the first opportunity to do these, otherwise I'll push a patch in a week or two.

**Comment by HowJMay** [comment]
Sorry for fixing late and thank you for helping as well

**Pull request merged**